modules/local/juicer_tools_pre.nf

```diff
@@ -0,0 +1,36 @@
+process JUICER_TOOLS_PRE {
+    tag "$meta.id"
+    label 'process_medium'
+
+//    conda (params.enable_conda ? "bioconda::java-jdk=8.0.112" : null)
+//    container "${ workflow.containerEngine == 'singularity' &&
+//                    !task.ext.singularity_pull_docker_container ?
+//        'https://depot.galaxyproject.org/singularity/java-jdk:8.0.112--1' :
+//        'quay.io/biocontainers/java-jdk:8.0.112--1' }"
```

Comment:
Same as above. I would leave java in if possible. The process may not run on a machine without java or consistently at least without this.